buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -18,6 +18,43 @@
         "cwd": "",
         "failOnStandardError": "false"
       }
+    },
+    {
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Initialize tools",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "pkg/init-tools.sh",
+        "arguments": "",
+        "workingFolder": "",
+        "failOnStandardError": "false"
```

Comment:
Taking a look at the results of a build, I saw that the inputs and the versionSpec don't match the build.sh shell script call step, so this doesn't work during the build.

```
  "versionSpec": "2.*",
  "definitionType": "task"
},
"inputs": {
  "scriptPath": "build.sh",
  "args": "$(BuildArguments)",
  "disableAutoCwd": "false",
  "cwd": "",
  "failOnStandardError": "false"
```

init-tools is a no-op, though, since build.sh already initialized tools. (When we need a version of the CLI/msbuild that we can use, though, we'll need this to actually pull down new tooling.)